huxley/www/static/js/huxley.browserify.js

```diff
@@ -35,9 +35,10 @@ var routes = (
     <Route path="/login" handler={LoginView} />
     <Route path="/password" handler={ForgotPasswordView} />
     <Route path="/password/reset" handler={PasswordResetSuccessView} />
-    <Route path="/register" handler={RegistrationView} />
+    <Route path="/register/open" handler={RegistrationView} />
```

Comment:
How about using the conference context here to conditionally assign the handler, so there's actually no way to get to the registration view if it's closed.